src/EA.Weee.Web/Areas/Scheme/Views/TransferEvidence/TransferEvidenceNote.cshtml

```diff
@@ -33,9 +33,32 @@
         $(document).ready(function () {
             $('#SelectAllCheckboxesSection').css("display", "block");
 
+            // case when selectAll is ticked => all checkboxes get selected, if selectAll is unticked => the other 14 checboxes will be unticked
```

Comment:
@meglenapnovoselska Is it possible to move this script into weee-application,js? so that it just gets included and doesnt have to be added to both views

Reply:
sure, yes. I will do it.